python/tk_multi_publish/publish.py

```diff
@@ -75,11 +64,14 @@ def work_template(self):
         """
         return self._app.get_template("template_work")
 
-    def rebuild_primary_output(self):
+    def build_outputs(self):
```

Comment:
could this rename break existing client/integration code? I would leave a legacy pathway here just to be sure....

Reply:
done!